### PR TITLE
SignatureSha256/FamilyName: Finebits.MetroCommander version 1.3.86.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24157)

### CODE REVIEW

File: manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml [hunk 1]
```diff
@@ -3,23 +3,24 @@ PackageVersion: 1.3.86.0
...
 PackageUrl: https://www.microsoft.com/store/apps/9WZDNCRFJB33
 Installers:
-- Architecture: neutral
+- Architecture: x64
   InstallerUrl: https://finebitsstorage.z22.web.core.windows.net/metrocommander/packages/MC_1.3.86.0.msix
   InstallerSha256: C30BC2FC1595D4A284C4350A63E7FD74DDDDA44C64D739DB54297B2C4E9D364A
   InstallerType: msix
+  SignatureSha256: 4375a570eda81875ad8fa4a3b0317b200c547a9e8169fe898dd2f7f28b53b93c
+  PackageFamilyName : BOOSTUDIOLLC.METROCOMMANDER_b6e429xa66pga
   InstallerSwitches:
```
Comment (ItzLevvie) [inline manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml:21]:
```suggestion
```
This isn't meant to be here.

File: manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml [hunk 2]
```diff
@@ -3,23 +3,24 @@ PackageVersion: 1.3.86.0
...
-    Silent: /S
-    SilentWithProgress: /S
+   Silent: /S
+   SilentWithProgress: /S
```
Comment (ItzLevvie) [inline manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml:23]:
```suggestion
```
This isn't meant to be here.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Comment by vedantmgoyal2009** [comment]
Yeah, I removed it first but then added it back as I thought Pipelines were failing due to this.

**Pull request closed without merging**